yark/archiver/config.py

```diff
@@ -44,6 +45,9 @@ class Config:
     comments: bool = False
     format: Optional[str] = None
     proxy: Optional[str] = None
+    bind_host: Optional[str] = None
+    bind_port: int = 7667
+    open_in_webbrowser: bool = True
```

Comment:
Can you rename this to `headless` and have it be False by default?

Reply:
renamed to headless, also changed the cli parameter to match